DOCSP-16494 consistent purpose statements (#103)
mongodb/docs-java: source/fundamentals/enterprise-auth.txt

   :depth: 2
   :class: singlecol

> In this guide, you can learn how to authenticate in the MongoDB Java driver using each

Reviewer: ```suggestion
In this guide, you can learn how to authenticate with MongoDB using each
```
Issue: I think this was correct before. I think you authenticate with the server (need to log into server), rather than with the driver.
